com.borgbase.vorta.yaml

```diff
@@ -0,0 +1,89 @@
+app-id: com.borgbase.vorta
+
+runtime: org.kde.Platform
+runtime-version: '5.12'
+sdk: org.kde.Sdk
+
+command: vorta
+
+finish-args:
+  # X11/Wayland
+  - --socket=x11
+  - --share=ipc
+  - --socket=wayland
+  - --device=dri
+  # Sync
+  - --share=network
+  # Filesystem access
+  - --filesystem=home
+  - --filesystem=/run/media
+  - --filesystem=/media
+  # dconf access
+  - --filesystem=xdg-run/dconf
+  - --filesystem=~/.config/dconf:ro
+  - --talk-name=ca.desrt.dconf
+  - --env=DCONF_USER_CONFIG_DIR=.config/dconf
+  # Tray icon
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --own-name=org.kde.StatusNotifierItem-2-1
+  # Mounting
+  - --talk-name=org.freedesktop.*
```

Comment:
What do you want to mount exactly here? Talking to the whole org.freedesktop is a bit too much. You can build the flatpak and run it with `flatpak run com.borgbase.vorta  --log-session-bus` which will tell you which bus you need to talk to and update manifest according to that.

Reply:
I want to mount a [borg repository as a FUSE filesystem](https://borgbackup.readthedocs.io/en/stable/usage/mount.html?highlight=mount#description)

I get the message `Portal call failed: org.freedesktop.DBus.Error.ServiceUnknown`.

Could you please tell me how to update the manifest?